app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -480,8 +517,11 @@ fun CreateActivityScreen(
                               location = selectedLocation,
                               images = items,
                               participants = attendees,
-                              type = types.find { it.name == selectedOption } ?: types[0],
-                              comments = listOf())
+                              type = types.find { it.name == selectedOptionType } ?: types[0],
```

Comment:
Is setting it to null really the best expected behaviour ? Shouldn't we force the user to choose a type instead ?